.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
`.eslintrc` is deprecated, use `.eslintrc.json` as the name of the file. Rename this in other projects too if you see it somewhere.